index.html

```diff
@@ -199,53 +199,53 @@ <h2>Let's Go For A Walk</h2>
-            </div>
-            <div class="form-group">
-              <label for="exampleInputEmail1">Email address</label>
-              <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
-            </div>
-            <div class="form-group">
-              <label for="exampleInputPassword1">Password</label>
-              <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
-            </div>
-            <div class="form-check">
-              <input type="checkbox" class="form-check-input" id="exampleCheck1">
-              <label class="form-check-label" for="exampleCheck1">I would like to receive promotional emails</label>
-              <br />
-              <input type="checkbox" class="form-check-input" id="exampleCheck2">
-              <label class="form-check-label" for="exampleCheck2">I would like to receive promotional emails</label>
-            </div>
-            <button type="submit" class="btn .readyForLaunch">Ready For Launch</button>
-          </form>
-        </div>
-        <div class="col-4">
-          <h5>Space Stuff</h5>
-          <ul class="list-group">
-            <li class="list-group-item">Space Helmet</li>
-            <li class="list-group-item">Rocket Boosters</li>
-            <li class="list-group-item">Shuttle</li>
-            <li class="list-group-item">Training for Launch</li>
-            <li class="list-group-item">Heat resistant plating</li>
-            <li class="list-group-item">Space Suit</li>
-          </ul>
-        </div>
+    </div>
+    <div class="row w-100 pb-5 pt-5">
+      <div class="col-8">
+        <h2>Ready To Go?</h2>
+        <form>
+          <div class="form-group">
+            <label for="exampleInputPassword1">Name</label>
+            <input type="name" class="form-control" id="inputName1" placeholder="name">
+          </div>
+          <div class="form-group">
+            <label for="exampleInputEmail1">Email address</label>
+            <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
+          </div>
+          <div class="form-group">
+            <label for="exampleInputPassword1">Password</label>
+            <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
+          </div>
+          <div class="form-check">
+            <input type="checkbox" class="form-check-input" id="exampleCheck1">
+            <label class="form-check-label" for="exampleCheck1">I would like to receive promotional emails</label>
+            <br />
+            <input type="checkbox" class="form-check-input" id="exampleCheck2">
+            <label class="form-check-label" for="exampleCheck2">I would like to receive promotional emails</label>
+          </div>
+          <button type="submit" class="btn .readyForLaunch">Ready For Launch</button>
```

Comment:
So we talked about this on Friday but you can create an id and change it's color. We would use id's in the case of using an element just once, which in this case, you are!

In your HTML: 
`<button type="submit" class="btn" id="readyForLaunch">Ready For Launch</button>`

CSS:
`#readyForLaunch { whatever u want }`

Try that out and see if it works for ya!